application-ckeditor-ui/src/main/resources/CKEditor/EditSheet.xml

```diff
@@ -295,6 +295,7 @@ define('xwiki-ckeditor', [
       // CKEditor uses '-' (dash) as locale separator (between the language code and the country code).
       language: currentLocale.toLowerCase().replace('_', '-'),
       uploadUrl: getUploadURL(sourceDocument, 'filetools'),
+      stylesSet: #if ($services.wysiwyg.HTMLSyntax.version.startsWith("5")) 'html5' #else 'html4' #end,
```

Comment:
I'm not a big fan of mixing JavaScript with Velocity, and in this case you can avoid it easily because you already set the ``data-syntax`` attribute from ``VelocityMacros``. So I'd rather use ``$(element).attr('data-syntax')`` or simply ``$(element).data('syntax')``.

Reply:
I do not think this is available in the inline editor. In https://github.com/xwiki-contrib/application-ckeditor/blob/6329cdec194cd7d36b6bdcf16a6f2508ebf9144a/application-ckeditor-ui/src/main/resources/CKEditor/InlineEditor.xml#L160-L164 this attribute is not added and I also couldn't see it using the developer tools.

Reply:
Indeed. You need to add it first to https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-edit/xwiki-platform-edit-ui/src/main/resources/XWiki/InplaceEditing.xml#L1587 so that you have it on ``config``. Then you need to modify the ``InlineEditor`` to take it from ``config`` and set it as container attribute, from where it will be read by ``EditSheet``.

Reply:
I have adapted this pull request and xwiki/xwiki-platform#1762 accordingly.